src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$accent-blue-color: #00acdc;
+$primary-title-color: rgba(6, 11, 53, 1);
+$primary-grey-color: rgba(97, 96, 112, 1);
+$primary-white-color: rgba(255, 255, 255, 1);
+$border-color: rgba(243, 243, 243, 1);
+$main-font: roboto, sans-serif;
```

Comment:
It's variable name don't tell me nothing, about what is font-family do you use. I think, you shouldn't use variable here, because it's below readability of the code